cmd/podman/manifest/add.go

```diff
@@ -13,7 +13,7 @@ import (
 var (
 	manifestAddOpts = entities.ManifestAddOptions{}
 	addCmd          = &cobra.Command{
-		Use:   "add",
+		Use:   "add [flags] LIST LIST",
```

Comment:
```suggestion
		Use:   "add [flags] LIST LIST",
```
Is `LIST LIST` correct? Shouldn't one be IMAGE?